x-pack/plugins/uptime/e2e/config.ts

```diff
@@ -44,6 +48,11 @@ async function config({ readConfigFile }: FtrConfigProviderContext) {
         `--elasticsearch.username=kibana_system`,
         `--elasticsearch.password=changeme`,
         '--xpack.reporting.enabled=false',
+        '--xpack.uptime.unsafe.service.manifestUrl=https://manifest.synthetics.elastic.dev/v1/manifest.json',
+        `--xpack.uptime.unsafe.service.username=${process.env.SYNTHETICS_SERVICE_USERNAME}`,
+        `--xpack.uptime.unsafe.service.password=${process.env.SYNTHETICS_SERVICE_PASSWORD}`,
+        '--xpack.uptime.unsafe.service.hosts=["https://synthetics-service-es.es.us-west2.gcp.elastic-cloud.com:9243"]',
```

Comment:
Trying to figure out how to manage the es instance so that the service can communicate back with the tests running on CI.